05/app.ts

```diff
@@ -0,0 +1,106 @@
+type CharacterType = 'Warrior' | 'Mage' | 'Healer';
+type CharacterSkill = 'Slam' | 'Fireball' | 'Heal';
```

Comment:
Pytanie czy nie podejść do tego trochę inaczej. Każda klasa wojownika ma swój specjalny atak. To nie są niezależne elementy jak w tym przypadku. Może lepiej stworzyć klasy potomne o konkretnych nazwach np. Warrior, Mage itp.